pkg/apis/storage/validation/validation.go

```diff
@@ -165,8 +167,39 @@ func validateAttacher(attacher string, fldPath *field.Path) field.ErrorList {
 // validateSource tests if the source is valid for VolumeAttachment.
 func validateVolumeAttachmentSource(source *storage.VolumeAttachmentSource, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	if source.PersistentVolumeName == nil || len(*source.PersistentVolumeName) == 0 {
-		allErrs = append(allErrs, field.Required(fldPath, ""))
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.CSIInlineVolume) {
```

Comment:
same. feature gate checking should be done only in *strategy.go when dropping fields

Reply:
See response in newer PR here - https://github.com/kubernetes/kubernetes/pull/74086#issuecomment-464188838

Reply:
See response in newer PR here - https://github.com/kubernetes/kubernetes/pull/74086#issuecomment-464188838